.hintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
- [ ] Your readme file is not descriptive 😨 Actually, You are not using the correct readme template 🤔 You can get the latest microverse readme template on this [link](https://github.com/microverseinc/readme-template) 👍